apps/web/src/pages/CommunityInterests/sections/FindANewCommunity.tsx

```diff
@@ -0,0 +1,283 @@
+import { useIntl } from "react-intl";
+import UserGroupIcon from "@heroicons/react/24/outline/UserGroupIcon";
+import { useFormContext } from "react-hook-form";
+import { ComponentProps } from "react";
+
+import { Heading, Well } from "@gc-digital-talent/ui";
+import { Checklist, RadioGroup, Select } from "@gc-digital-talent/forms";
+import {
+  commonMessages,
+  errorMessages,
+  uiMessages,
+} from "@gc-digital-talent/i18n";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+
+import { FormValues } from "../form";
+
+export const FindANewCommunityOptions_Fragment = graphql(/* GraphQL */ `
```

Comment:
exported declaration `FindANewCommunityOptions_Fragment` not used within other modules
```suggestion
const FindANewCommunityOptions_Fragment = graphql(/* GraphQL */ `
```